docs/content/latest/explore/ysql-language-features/going-beyond-sql/follower-reads-ysql.md

```diff
@@ -27,11 +27,27 @@ showAsideToc: true
   </li>
 </ul>
 
-## Introduction
+## Leader lease mechanism
+
+In a distributed environment, when one node in a cluster is elected as the leader holding the latest data, it is possible that another node may assume that it is the leader, and that it holds the latest data. This could result in serving stale reads to a client. To avoid this confusion, YugabyteDB provides a **leader lease** mechanism  where an elected node member is guaranteed to be the leader until it's lease expires.
```

Comment:
```suggestion
In a distributed environment, when one node in a cluster is elected as the leader holding the latest data, it is possible that another node may assume that it is the leader, and that it holds the latest data. This could result in serving stale reads to a client. To avoid this confusion, YugabyteDB provides a _leader lease_ mechanism  where an elected node member is guaranteed to be the leader until its lease expires.
```